application/controllers/template_controller.py

```diff
@@ -25,14 +25,23 @@ def _get_template_by_id(template_id):
     return template
 
 
-def get_templates_by_classifiers(classifiers):
+def get_template_by_classifiers(classifiers):
     try:
-        templates = db.session.query(CommunicationTemplate).filter(CommunicationTemplate.classification == classifiers)\
-            .all()
+        template = db.session.query(CommunicationTemplate).filter(CommunicationTemplate.classification == classifiers)\
+            .first()
     except SQLAlchemyError:
         logger.exception('Unable to retrieve template with classifiers', classifiers=classifiers)
         raise DatabaseError(f'Unable to retrieve template with classifiers: {json.dumps(classifiers)}', status_code=500)
-    return templates
+    return template
+
+
+def get_comms_template_by_classifiers(classifiers=None):
+    template = get_template_by_classifiers(classifiers)
```

Comment:
I'm assuming DatabaseError is handled higher up?

Reply:
In the flask error handlers.